vocos/discriminators.py

```diff
@@ -1,11 +1,14 @@
-from typing import Tuple, List
+from typing import List, Optional, Tuple
 
 import torch
 from torch import nn
 from torch.nn import Conv2d
 from torch.nn.utils import weight_norm
 
 
+PeriodsType = Tuple[int, int, int, int, int]
```

Comment:
This might be of variable size

Reply:
Sorry for not replying before. I just fixed this.